src/commands/hld/pipeline.ts

```diff
@@ -83,6 +85,29 @@ export const execute = async (
 ) => {
   try {
     populateValues(opts);
+    const accessOpts: IAzureDevOpsOpts = {
+      orgName: opts.orgName,
+      personalAccessToken: opts.personalAccessToken,
+      project: opts.devopsProject
+    };
+
+    // By default the version descriptor is for the master branch
+    const hasPipelineFile = await repositoryHasFile(
+      RENDER_HLD_PIPELINE_FILENAME,
+      opts.yamlFileBranch ? opts.yamlFileBranch : "master",
+      opts.hldName,
+      accessOpts
+    );
+
+    if (!hasPipelineFile) {
+      logger.error(
+        "Error installing build pipeline. Repository does not have a " +
+          RENDER_HLD_PIPELINE_FILENAME +
+          " file."
+      );
+      await exitFn(1);
+    }
```

Comment:
you need a else here that's
```
if (!hasPipelineFile) {
...
} else {
   await installHldToManifestPipeline(opts);
   await exitFn(0);
}
```

otherwise exitFn will be called twice.